.github/ISSUE_TEMPLATE/1.bug.yml

```diff
@@ -9,7 +9,7 @@ body:
     attributes:
       label: Search keywords
       description: |
-        Your issue may have already been reported! First search for duplicates among the [existing issues](https://github.com/mui/mui-x/issues).
+        Your issue may have already been reported! First search for duplicates among the [existing issues](https://github.com/mui/mui-x/issues?q=is%3Aopen+is%3Aclosed).
```

Comment:
I think that we are missing the extra `is:issue` argument in all of these instances.
By going to `/issues` both `is:issue` and `is:open` filters are applied.

Reply:
This is a bit of a generalistic question. 

1. PRs on github are also issues and `is:issue` would exclude them: Do we want to include PRs in this search or not? 🤷🏼 
2. `is:open` will not show any closed issues, but since the issue might be already asked/solved we absolutely want to have them in this list.

Reply:
But PRs generally target resolving an issue.
I'm not sure we should be polluting the list of searchable entries by including PRs. 🤷 
I just raised the point, that this technically causes a "regression" as compared to the existing approach.
If we actually want to include PRs as well, then sure, all is good, but I didn't see anything mentioned about this in the description, hence the flagging. 😉

Reply:
I have no strong opinion on incuding PRs or not ... Most of the time the search should be accompanied by a search term to limit the results anyways, so the PRs might not be as prominent.

On the other hand you are right about the purpose of PRs in general. There might be additional information in them that could sometimes lead the user to a solution they are looking for, but there are too many ifs and buts in this.

I will add the `is:issue` parameter

Reply:
Or rather: I will change this to https://github.com/mui/mui-x/issues?q=is%3Aissue

This will naturally include all issue states! 👍🏼